go/libraries/doltcore/doltdb/root_val.go

```diff
@@ -70,6 +70,10 @@ type RootValue interface {
 	GetCollation(ctx context.Context) (schema.Collation, error)
 	// GetRootObject will retrieve a root object by its case-sensitive name.
 	GetRootObject(ctx context.Context, objName TableName) (RootObject, bool, error)
+	// GetRootObjectConflicts will retrieve a conflict root object by its case-sensitive name.
+	GetRootObjectConflicts(ctx context.Context, tName TableName) (ConflictRootObject, bool, error)
+	// GetRootObjectsWithConflicts retrieves all root objects that have conflicts.
+	GetRootObjectsWithConflicts(ctx context.Context) ([]RootObject, error)
```

Comment:
How about `GetConflictRootObject()` to be more consistent with the name of the type being returned?

It seems like the second function returns RootObjects that are NOT ConflictRootObjects themselves, right? and then the caller has to separately go query for the `ConflictRootObject` instance?